src/GraphQL.NewtonsoftJson/OperationMessageJsonConverter.cs

```diff
@@ -23,9 +23,9 @@ public class OperationMessageJsonConverter : JsonConverter
         public override bool CanWrite => true;
 
         /// <inheritdoc/>
-        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        public override void WriteJson(JsonWriter writer, object? value, JsonSerializer serializer)
```

Comment:
If they marked `value` as nullable then they suppose `WriteJson` _may_ accept `null` so we have NRE on line 30.

Reply:
Could be... I can add a check in any case.